src/main/java/seedu/tasklist/TaskList.java

```diff
@@ -185,4 +185,4 @@ public static int categoryCounter(LocalDate checkDate) {
         }
         return totalTasksDay;
     }
-}
+}
```

Comment:
There should be a new line at the end of all code to adhere to git standards

Reply:
Okay~ I haven't noticed it.
{
Need I delete it?


}